scripts/omDomainJSON.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright 2023 The Superpower Institute Ltd.
+#
+# This file is part of OpenMethane.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Generate a JSON file describing the domain and grid.
+"""
+
+import numpy as np
+import netCDF4 as nc
+from openmethane_prior.omInputs import domainXr, domainProj
+from openmethane_prior.omOutputs import domainJSONOutputPath
+import json
+
+def make_point(x, y):
+    return ([float(x), float(y)])
+
+def write_domain_json(output_file):
+    # Load raster land-use data
+    print("converting domain grid details to JSON")
+
+    domain = {
+        "crs": {
+            "projection_type": "lambert_conformal_conic",
+            "standard_parallel": float(domainXr.attrs['TRUELAT1']),
```

Comment:
Why the extra cast to float? Are these sometimes integers?

Reply:
I think they might actually have **higher** precision than what JSON supports, but don't quote me on that. This made Python happy so I just went with it.